mapApp/utils/weather.py

```diff
@@ -9,7 +9,7 @@
 # Removed test to see if rider was travelling in the same direction in favor of storing the cardinal wind direction only
 # Updated docstring
 
-import urllib2
+import urllib.request, urllib.error, urllib.parse
```

Comment:
I don't see urllib.error or urllib.parse being used in the code. Are these imports necessary?